dist/index.html

```diff
@@ -4,32 +4,30 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link
-    rel="stylesheet"
-    href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
-    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
-    crossorigin="anonymous"
-  />
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <title>API based WebApp</title>
+    <title>API-based-webapp</title>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+
 <script defer src="main.bundle.js"></script></head>
 <body>
-    <header class="header-container">
-        <nav class="navigation">
-            <a href="#" class="my-logo">AuctionArt</a>
-            <a class="menu-bars"><i class="fal fa-bars"></i></a>
-            <ul class="nav-list">
-                <li>Exhibition</li>
-                <li>About</li>
-                <li>Blog & News</li>
-            </ul>
-        </nav>
-    </header>
-    <main class="main-class">
-        <section class="home-page"></section>
-    </main>
-    <footer>
-        <p class="my-footer">Created with &#10084;&#65039; by Roland M</p>
-    </footer>
+  <header>
+    <div class="header-wrapper">
+    <div class="logo">
+      <a href="#" class="logo">
+        <img src="../src/images/nasa2.png" alt="logo">
+      </a>
+    </div>
+    <nav>
+      <ul>
+        <li class="nav-link" id="item-count"><a href="#">APOD</a></li>
+        <li class="nav-link"><a href="#">Earth Images</a></li>
+        <li class="nav-link"><a href="#">Moon Images</a></li>
+      </ul>
```

Comment:
- [ ] Counters should be present in all the interfaces that show the number of items (home), the number of comments (comments popup). Let's add the count on the Home navbar page.

Reply:
- [ ] As the previous reviewer said, you should show the counter of items that you are showing on your home page and for comments too.

Reply:
- [ ] As previous reviewers have said, you must implement a counter with the total of items that load on the page. I
I'm aware of your situation, but I'm also know that one of the partners had to implement counters on items and the other in comments. Unfortunately, I don't see either.